src/sequence-steps.js

```diff
@@ -279,7 +248,10 @@ export function gitPushOriginMaster( [ git, options ] ) {
 export function githubUpstream( [ git, options ] ) {
 	const command = `git config remote.upstream.url`;
 	return utils.exec( command ).then( data => {
-		const [ , owner, name ] = data.match( /github.com[:/](.*)\/(.*)\.git/ ) || [];
+		let [ , owner, name ] = data.match( /github.com[:/](.*)\/(.*)/ ) || []; // eslint-disable-line prefer-const
```

Comment:
In what case were you getting `.git` in the name? I'd rather update the regex to fix that instead of adding another statement to filter that out. Do you have an example of a URL that was breaking the original code?